CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Welcome to Scaffold-ETH 2 Contributing Guide 
+
+Thank you for investing your time in contributing to Scaffold-Eth 2!  
+
+This guide aims to provide overview of the contribution workflow to help us make the contribution process effective for everyone involved. 
+
+## Table of Contents
+1. [About the Project](#about-the-project)
+1. [Project Status](#project-status)
+1. [Getting Started](#getting-started)
```

Comment:
The numbers are off here and also we are not using numbers on the Sections.

We could either update the numbers here (and add them to the Sections titles) or remove them completely.